clusterfilehandlers/dfsbackends/oracle.php

```diff
@@ -1269,14 +1255,19 @@ protected function _fail( $value, $text = false )
         return new eZMySQLBackendError( $value, $text );
     }
 
+    protected function _queryAndCommit( $query, $fname = false, $reportError = true, $bindparams = array(), $return_type = self::RETURN_BOOL )
```

Comment:
yes. I wanted to keep the callers of this as clean as possible, and there's already too many params in the middle with default values. The name of the method is also a good hint for when you find it used in code